### PR TITLE
Update the date_invited field when resending invites.

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
This will resolve #35 by displaying the last time an invitation was sent for a particular invite, rather than always showing the date of the _original_ invitation. ping @ldko, @clarktr1, this is ready for review.

File: tests/test_all.py [hunk 1]
```diff
@@ -480,17 +480,24 @@ def test_forgotten_password(self):
...
             reverse('invite:resend', args=[code]),
             follow=True
         )
+        self.alpha_invite.refresh_from_db()
         self.assertEqual(200, response.status_code)
         self.assertIn(
             'Resent invitation email for {}'.format(self.alpha_invite.username),
             response.content.decode()
         )
+        self.alpha_invite.refresh_from_db()
```
Comment (ldko) [inline tests/test_all.py:498]: Do we need to `refresh_from_db` again here after doing it a few lines up?

**Review by clarktr1 — APPROVED** [review]
Looks good and tests work for me!

File: tests/test_all.py (hunk 1 above)
Comment (somexpert) [reply tests/test_all.py:498]: heh, nope, that was a mistake on my part, I'll fix it

**Review by ldko — APPROVED** [review]
Works for me. Thanks @somexpert :+1:

**Pull request merged**